ocaml/idl/schematest.ml

```diff
@@ -0,0 +1,72 @@
+let hash x = Digest.string x |> Digest.to_hex
+
+(* (md5 hash, [(major, minor), ...]) *)
+let hash_to_version = [("f3b7df44ff977a0243abc6742b64c418", [(5, 700)])]
+
+let no_bump hash =
+  Printf.sprintf {|("%s", [(%i, %i)])|} hash Datamodel_common.schema_major_vsn
+    Datamodel_common.schema_minor_vsn
+
+let minor_bump hash =
+  Printf.sprintf {|("%s", [(%i, %i)])|} hash Datamodel_common.schema_major_vsn
+    (Datamodel_common.schema_minor_vsn + 1)
+
+let major_bump hash =
+  Printf.sprintf {|("%s", [(%i, %i)])|} hash
+    (Datamodel_common.schema_major_vsn + 1)
+    0
+
+let current_schema_hash : string =
+  let hash_of_obj x =
+    Datamodel_types.rpc_of_obj x |> Jsonrpc.to_string |> hash
```

Comment:
We only need to look at the `contents` of an object (the fields). You don't need a new schema version if you're just adding a function.